tutorials/premake5.lua

```diff
@@ -29,7 +29,14 @@ function copydir(src_dir, dst_dir, filter, single_dst_dir)
     end
 end
 
+
+
 workspace "hiprtSdkTutorial"
+
+    include("../build/conanbuildinfo.premake.lua")
```

Comment:
We just need to add this in premake to consume all dependencies, including Orochi.